clinical-clusters/2018/clinical-clusters.json

```diff
@@ -2392,22 +2365,6 @@
       }
     ]
   },
-  {
-    "measureId": "134",
-    "submissionMethod": "registry",
-    "firstPerformanceYear": 2017,
-    "lastPerformanceYear": null,
-    "clinicalClusters": [
-      {
-        "name": "palliativeCare",
-        "measureIds": [
-          "047",
-          "134",
```

Comment:
same q here

Reply:
Same thing here... the requirement for the "palliativeCare" cluster: "Note: When submitting #47 and/or #134, they are not subject to EMA for these clinically related measures. If #342 is submitted, it is expected to submit quality measures #47 and #134. "